### PR TITLE
feat(spanner/spansql): support for parsing a DML file

### PR DESCRIPTION
**Pull request by 110y**
- Add `spansql.ParseDML` corresponding to existing `ParseDDL` in order to support for parsing a DML file.

### Background

As I am a maintainer of the [`cloudspannerecosystem/wrench`](https://github.com/cloudspannerecosystem/wrench/), which is a schema migration tool for Spanner, I'm planning to use `spansql` in order to replace the current regex-based parsing. `wrench` supports executing the multiple DMLs in a file, so I'd like `spansql` to have the `ParseDML` function same as the  `ParseDDL` function.

### Note

This PR does not add support for `Position` of DMLs. It should be added as another PR.

### CODE REVIEW

File: spanner/spansql/parser_test.go [hunk 1]
```diff
@@ -32,7 +32,8 @@ func TestParseQuery(t *testing.T) {
 		want Query
 	}{
 		{`SELECT 17`, Query{Select: Select{List: []Expr{IntegerLiteral(17)}}}},
-		{`SELECT Alias AS aka From Characters WHERE Age < @ageLimit AND Alias IS NOT NULL ORDER BY Age DESC LIMIT @limit OFFSET 3` + "\n\t",
+		{
```
Comment (110y) [inline spanner/spansql/parser_test.go:35]: These formatting changes were made by `gopls` with its `gofumpt` option.

**Comment by zchee** [comment]
@rahul2393 How about this PR review progress?

**Comment by rahul2393** [comment]
Hello, @zchee this PR will be released in next release, review is in progress.

**Comment by zchee** [comment]
@rahul2393 Thanks! can't wait to this feature :D

**Comment by 110y** [comment]
The build was failed due to [`spanner/spansql/parser.go:79:1: exported function ParseDML should have comment or be unexported`](https://source.cloud.google.com/results/invocations/ce563894-f29a-4621-9822-2f40f47159e4/targets/cloud-devrel%2Fclient-libraries%2Fgo%2Fgoogle-cloud-go%2Fpresubmit%2Flatest-version/log). 
So, I just added comments to the `ParseDML` function.

@rahul2393 Could you please check and rerun the build?

**Review by rahul2393 — APPROVED** [review]

**Pull request merged**